release_dashboard/lib/widgets/clean_release.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../services/conductor.dart';
+import 'package:flutter/material.dart';
+
+import 'common/dialog_prompt.dart';
+import 'common/snackbar_prompt.dart';
+
+/// Button to clean the current release.
+///
+/// When the button is clicked, a dialogue prompt will open for confirmation.
+/// Clicking on [Yes] will clean the release, [No] will close the dialogue prompt.
+class CleanRelease extends StatefulWidget {
```

Comment:
If this is a button, you should make it `CleanReleaseButton` (and make the file match the scheme)